src/app/views/cheat-sheets/science/science.component.html

```diff
@@ -118,7 +118,7 @@
         <div class="col-12 col-lg-5">
             <div class="eq-container">
                 <div class="eq-field bg-factorio-green">
-                    <span class="eq-label">Research cycle time</span>
+                    <span class="eq-label">Research cycle time (seconds)</span>
```

Comment:
Need to keep these short, maybe just "(s)" or "in s" or get rid of time
```suggestion
                    <span class="eq-label">Research cycle seconds</span>
```